tilequeue/log.py

```diff
@@ -357,6 +360,9 @@ def _log(self, msg, parent, pyramid=None, coord=None):
             json_obj['pyramid'] = make_coord_dict(pyramid)
         if coord:
             json_obj['coord'] = make_coord_dict(coord)
+        if coord_start_ms is not None:
```

Comment:
do you need the `is not None`?  Looks like the case right above this doesn't use it.  No biggie either way

Reply:
> do you need the `is not None`? Looks like the case right above this doesn't use it. No biggie either way

since this field is integer, we also want to consider if the value is 0, that's why we cannot use the simple version

Reply:
sounds good